src/scripts/episodes.js

```diff
@@ -37,15 +37,21 @@ define(["loading", "events", "libraryBrowser", "imageLoader", "listView", "cardB
 
         function reloadItems(page) {
             loading.show();
+            isLoading = true;
             var query = getQuery(page);
             ApiClient.getItems(Dashboard.getCurrentUserId(), query).then(function(result) {
                 function onNextPageClick() {
-                    query.StartIndex += query.Limit, reloadItems(tabContent)
+                    if (isLoading) return;
```

Comment:
> > @JustAMan I looked at it but it's in bower_components/emby-webcomponents/loading and I don't know if it's alright to modify there, don't feel like making conflicts and stuff but if I could, I would add a getter to the loader such as loader.visible(), or loader.active()
> 
> I'm all for making this a separate lib, but let's finish this PR first.
> As for that `loader` being somewhere else - what's the issue with it? I think it's fine to modify, and it should be better to modify it rather than stick a separate flag everywhere.

Cool, I'll go for that then

Reply:
A future improvement might be pre-loading prev and next pages.

Reply:
I actually like both having an ability to choose N and to keep prev and next pages preloaded.